offchain/authority-claimer/src/config/cli.rs

```diff
@@ -0,0 +1,138 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+use clap::{command, Parser};
+use eth_tx_manager::{
+    config::{TxEnvCLIConfig as TxManagerCLIConfig, TxManagerConfig},
+    Priority,
+};
+use rollups_events::{BrokerCLIConfig, BrokerConfig};
+use rusoto_core::Region;
+use snafu::ResultExt;
+use std::{fs, path::PathBuf, str::FromStr};
+
+use crate::config::{
+    error::{
+        AuthorityClaimerConfigError, InvalidRegionSnafu, MnemonicFileSnafu,
+        TxManagerSnafu, TxSigningConfigError, TxSigningSnafu,
+    },
+    json::{read_json_file, DappDeployment},
+    AuthorityClaimerConfig, TxSigningConfig,
+};
+
+// ------------------------------------------------------------------------------------------------
+// AuthorityClaimerCLI
+// ------------------------------------------------------------------------------------------------
+
+#[derive(Clone, Parser)]
+#[command(name = "rd_config")]
+#[command(about = "Configuration for rollups authority claimer")]
+pub(crate) struct AuthorityClaimerCLI {
```

Comment:
[Clippy](https://rust-lang.github.io/rust-clippy/master/index.html#/upper_case_acronyms) will probably complain about the uppercase CLI. The same applies to the other CLI structs.

```suggestion
pub(crate) struct AuthorityClaimerCli {
```

Reply:
No problem, then 😉